packages/@webex/plugin-meetings/test/unit/spec/annotation/annotation.ts

```diff
@@ -0,0 +1,349 @@
+import MockWebex from '@webex/test-helper-mock-webex';
+import MockWebSocket from '@webex/test-helper-mock-web-socket';
+import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+import Mercury from '@webex/internal-plugin-mercury';
+import LLMChannel from '@webex/internal-plugin-llm';
+
+import AnnotationService from '../../../../src/annotation/index';
+import {ANNOTATION_RELAY_TYPES, ANNOTATION_REQUEST_TYPE, EVENT_TRIGGERS} from '../../../../src/annotation/constants';
+
+
+describe('live-annotation', () => {
+  const locusUrl = 'https://locus.wbx2.com/locus/api/v1/loci/163c1787-c1f5-47cc-95eb-ab2d660999e6';
+
+  describe('annotation', () => {
+    let webex, annotationService;
+
+    beforeEach(() => {
+      webex = new MockWebex({
+        children: {
+          mercury: Mercury,
+          llm: LLMChannel,
+          annotation: AnnotationService,
+        },
+      });
+
+
+      annotationService = webex.internal.annotation;
+      annotationService.connect = sinon.stub().resolves(true);
+      annotationService.webex.internal.llm.isConnected = sinon.stub().returns(true);
+      annotationService.webex.internal.llm.getBinding = sinon.stub().returns(undefined);
+      annotationService.webex.internal.llm.getLocusUrl = sinon.stub().returns(locusUrl);
+      annotationService.approvalUrl = 'url/approval';
+      annotationService.locusUrl = locusUrl;
+
+
+      webex.request = sinon.stub().returns(Promise.resolve('REQUEST_RETURN_VALUE'));
+      annotationService.register = sinon.stub().resolves({
+        body: {
+          binding: 'binding',
+          webSocketUrl: 'url',
+        },
+      });
+    });
+
+
+    describe('#MessageProcess', () => {
```

Comment:
issue: The convention is to use `#` to indicate that this is a test for a specific method/attribute for a class. i.e. you could have `describe('#eventCommandProcessor())`

Or you could still group the tests as you have done but remove the `#` i.e. `describe('message processing')`

Reply:
got it